server/controllers/index.js

```diff
@@ -1,9 +1,54 @@
 const models = require('../models');
 
 module.exports = {
-  sayHello: (req, res) => {
-    const something = models.sayHello();
-    res.sendStatus(200);
-    res.send(something);
+  getReviews: (req, res) => {
+    // console.log(req.query);
+    const {
+      product_id, page, count, sort,
+    } = req.query;
+    models.getReviewsByProduct(product_id, page, count, sort, (err, data) => {
+      if (err) {
+        res.status(404).send(err);
+      } else {
+        res.status(200).json(data);
+      }
+    });
+  },
+  getMetaData: (req, res) => {
+    const productId = req.query.product_id;
+    models.getMetaData(productId, (err, data) => {
+      if (err) {
+        res.status(400).send(err);
```

Comment:
is there a reason why you use status code 400 here? just curious since i don't know too much about status code.

Reply:
i do not know. I just put it there :)